specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,1430 @@
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskKind"
+        }
+      }
+    },
+    "AnalyzeConversationTaskResult": {
+      "description": "The base class of a conversation task result.",
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskResultsKind"
+        }
+      }
+    },
+    "CustomConversationalTask": {
+      "description": "The input for a custom conversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTask"
+        }
+      ],
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/ConversationAnalysisOptions"
+        },
+        "parameters": {
+          "$ref": "#/definitions/CustomConversationTaskParameters"
+        }
+      },
+      "x-ms-discriminator-value": "CustomConversation",
+      "required": [
+        "analysisInput",
+        "parameters"
+      ]
+    },
+    "CustomConversationTaskParameters": {
+      "type": "object",
+      "description": "Input parameters necessary for a Custom Conversation Task.",
+      "properties": {
+        "projectName": {
+          "type": "string",
+          "description": "The name of the project to use."
+        },
+        "deploymentName": {
+          "type": "string",
+          "description": "The name of the specific deployment of the project to use."
+        },
+        "verbose": {
+          "type": "boolean",
+          "description": "If true, the service will return more detailed information in the response."
+        }
+      },
+      "required": [
+        "projectName",
+        "deploymentName"
+      ]
+    },
+    "CustomConversationalTaskResult": {
+      "description": "The results of a custom conversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTaskResult"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "results": {
+          "$ref": "#/definitions/AnalyzeConversationResult"
+        }
+      },
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "CustomConversationResult"
+    },
+    "ConversationItemBase": {
+      "type": "object",
+      "discriminator": "modality",
+      "description": "The abstract base for a user input formatted conversation (e.g., Text, Transcript).",
+      "properties": {
+        "participantId": {
+          "description": "The user input unique identifier corresponding each participant in the conversation",
+          "type": "string"
+        },
+        "id": {
+          "description": "The user input unique identifier corresponding the conversation Item (utterance)",
+          "type": "string"
+        },
+        "language": {
+          "description": "The input language of the conversation item",
```

Comment:
Is this BCP-47? If so, please say that so customers know what valid inputs look like. If not, please also clarify what the expected format is. "language" is rather vague.

Reply:
Yes it is. Updated the description.